inference-engine/tests/functional/plugin/gna/pass_tests/remove_permutations_NHWC_to_NCHW_pass.cpp

```diff
@@ -86,10 +86,53 @@ class RemovePermutationsNHWCToNCHWPassTest : public testing::WithParamInterface<
         }
 };
 
+class RemovePermutationsNHWCToNCHWPass4DOutputTest : public testing::WithParamInterface<removePermutationsPassParams>,
```

Comment:
does this related to 4D export? 
I expected see parameteric test with 2d and 4d outputs while ngraph builder might conditionally insert reshape layer at the end to match test target  layout,